pkg/click/click

```diff
@@ -206,6 +221,10 @@ DEPS=$*
 if [[ $KHAN_TED -ne 0 ]]; then
     CLICK_CMD="$CLICK_CMD -k"
 fi
+# setup shared variables
+if [[ $CARD -ne 0 ]]; then
+    CLICK_CMD=$HOON
+fi
```

Comment:
Aside from using `CLICK_CMD` for something different than it's intended, this isn't necessary since `CLICK_CMD` isn't used at all if the `-c` option is enabled

Reply:
Removed this. I think it was remenet of some testing I did